health_checks.js

```diff
@@ -121,10 +124,25 @@ function formatError(code, exception, type) {
     return errorObject;
 }
 
+/**
+ * 
+ * @param {*} error 
+ * @param {*} param1 - S3 putObject parameters
+ * @param {*} callback 
+ * @returns 
+ */
+function handleIngestEncodingInvalidError(err, { data, key, bucketName }, callback) {
+    if (err.httpErrorCode === INGEST_INVALID_ENCODING.code && err.message.includes(INGEST_INVALID_ENCODING.message)) {
```

Comment:
I suggest we use error code only, because error message can be changed in future.

Reply:
I will remove